### DISCUSSION TITLE
Bring back the latency value for the last turn of dead snakes.

### DISCUSSION THREAD

**Discussion by smallsco** [BattlesnakeOfficial/feedback#91] [Bug Reports]
The new board design looks great, but there's a big UX regression that affects my testing process - it's no longer possible to tell if a snake died from timing out.

Previously, when a snake died, the latency from the turn it died would be shown for the remainder of the game. If this latency was from a big red 0, I would immediately know that there was a timeout. But if there was a black number > 0, I would immediately know that my algorithm returned an incorrect move.

Now, however, when a snake dies its' latency is no longer shown, making it impossible to tell. So any time I die in a strange way, I need to manually download the JSON for the turn and test it locally, whereas previously I'd only do that on games where I didn't timeout.

Please bring back the latency values for the last turn of dead snakes!

**Comment by bvanvugt** [comment]
Oof, this is an excellent point. I've added it back (along with author) in this commit:
https://github.com/BattlesnakeOfficial/board/commit/5bf744330affa2eb46ef194c8c175ddd49fba468

This will go live shortly in Board v1.0.37.
